src/pages/EditOrganization.tsx

```diff
@@ -142,6 +150,19 @@ const UsersTable = () => {
               }
             </Td>
             <Td isNumeric>{orgUser.userId}</Td>
+            <Td>
+              <Flex justifyContent = "flex-end">
+                <Button colorScheme = "green">
```

Comment:
If this confirm button is not doing anything we should probably remove it from this PR.

Reply:
Agreed. A confirmation popover would be sufficient, then I think we can approve this branch for master.